### ISSUE TITLE
Not sure its an issue, but want some insight!

### ISSUE THREAD

**Issue by fixingthingsguy** [MajicDesigns/MD_TCS230#2]
Hello: I've been using your code and it has worked well for my needs.
So, now I've added a feature to my project to use a SG90 servo.(rotate a disc with colors, read the color)  I also have used the ServoTimer2 library to avoid any timer conflicts.
However, when I do "readsensor()", (readsensor is function from your simple examples) the servo motor does not appear to be activated (does not turn). The sensor works fine at the expense of the
motor turning! If I remove the "readsensor"  the SG90 seems to work as designed but the sensor doesn't display. Conceptual code below
```
void loop () {
turn 90 degrees; delay(1000); readsensor();
}
```
Any insight appreciated. Thanks for your time.
Regards

**Comment by MajicDesigns** [comment]
Sorry, no idea why this is happening. Usually resolving the timer conflict fixes any problems.

**Comment by MajicDesigns** [comment]
A thought. Have you tried making the delay longer (ie, really cover the entire movement period for the servo). Does that make it work better?

**Comment by fixingthingsguy** [comment]
I tried many scenarios a bit haphazardly and I need to go through it more methodically.
I'm able to print the servo motor values at the time the write takes place, but the motor itself doesn't turn. I have it powered from another standalone Arduino with the ground shared with the project Arduino and the servo (SG90) gets 4.8vdc which should be enough, for hardly any load at all. 
I will try more methodically giving the md.tcs230 read more time as well and report back.
There is some other data I researched where apparent timer conflicts were removed with the use of the servotimer2 library. The problem I'm seeing is similar to that reported in an Arduino forum  http://forum.arduino.cc/index.php?topic=197818.0  Unfortunately the writer didn't wrap up so we don't know if he got it working.
Again, I'm quite sure it has nothing to do with the md.tcs230 libraries.

**Comment by MajicDesigns** [comment]
As per the forum discussion, have you tried moving the servo manually to a different position? Once it moves to 90 degrees, then it will appear to not move after that.

**Comment by fixingthingsguy** [comment]
Yes, I noticed that when the servo gets to the end of the rotation, it doesn't swing back, so manually resetting for the moment
As a lark I introduced a digitalwrite after the servo write. That now enables the servo to move as required.
With a 4 sec delay for the "readsensor:" I get the RGB(wait its not good) readings and the servo moving as required. Kind of fussy to set the delays. But the RGB data is 0,0,0. So I removed the servo code and ran and this time the RGB values make sense (not zeros).  All this makes me uncomfortable as I don't know whats going on. I will double check the wiring connections but I'm not sure that's the problem.
I of course don't know the operation of these timers, so maybe the use of the timer by freq.h conflicts with servotimer2 . 
I don't know. I'm going to back off a while and do some more research. Thanks for your suggestion.

**Comment by fixingthingsguy** [comment]
Thanks for allowing me to post. I wanted to close out by saying that I got rid of servotimer2 for my needs.
I achieved the rotation of a disc with color matching (prototype) by
a)fixing the sg90 servo so it gets to go around 360 (http://www.instructables.com/id/How-to-Make-a-TowerPro-Micro-Servo-Spin-360/) ; one of many hacks
b)removed the servotimer2.h
c)replaced the commands for turning with something very basic, ie using digitalWrite (http://www.robotshop.com/blog/en/arduino-5-minute-tutorials-lesson-5-servo-motors-3636)
So, what I got is a disc rotating, controlled by md.tcs230 /freqcount that as the disc rotates reads off the colors on the disc.
OF course not perfected, but I wasn't able to get this far before. So all is well with md.tcs230 and servotimer2 was not needed.
